src/modules/edc-demo/models/http-datasink-properties.ts

```diff
@@ -0,0 +1,41 @@
+export interface HttpDatasinkProperties {
```

Comment:
No need for this class, rename the original "HttpDatasourceProperties" to "HttpRequestParams". This should be done, because the backend uses one class for both http datasource and http datasink properties.